client/user/src/mealorder/meal-order-client-app.html

```diff
@@ -0,0 +1,306 @@
+                                        </div>
+                                    </div>
+                                </div>
+
+                                <div name="history">
+                                    <div id="menu-history" class="route-title"><a href="/#/menu/">Сделать
+                                        заказ</a>
+                                    </div>
+                                    <div class="icon-container">
+                                        <a href="/#/menu/">
+                                            <iron-icon class="history-icon"
+                                                       icon="maps:icons:add"></iron-icon>
+                                        </a>
+                                        <div>
+                                            <iron-icon class="app-toolbar-content" icon="icons:date-range"
+                                                       drawer-toggle></iron-icon>
+                                        </div>
+                                    </div>
+                                </div>
+                            </iron-pages>
+                        </div>
+                    </app-toolbar>
+                </app-header>
+                <iron-pages
+                        selected="[[routeData.page]]"
+                        attr-for-selected="name"
+                        fallback-selection="view404"
+                        role="main">
+                    <meal-order-menu-container name="menu"
+                                           on-dish-added="_addDishToOrder"></meal-order-menu-container>
+                </iron-pages>
+
+            </app-header-layout>
+
+        </app-drawer-layout>
+    </template>
+    <script>
+        /**
+         * Main user meal order component.
+         *
+         * <p>Is adopted for both desktop and mobile.
+         * Consists of 2 page:
+         * 1. A page with vendor menus, and user's shopping cart.
+         * 2. A page with user's history.
+         *
+         * @customElement
+         * @polymer
+         * @appliesMixin ExlibrisApplication.ReduxMixin
+         */
+        class MealOrderClientApp extends MealOrderApp.ReduxMixin(Polymer.Element) {
+            static get is() {
+                return 'meal-order-client-app';
+            }
+
+            static get properties() {
+                return {
+                    /**
+                     * Used for routing.
+                     */
+                    routeData: {},
+
+                    /**
+                     * Chosen date.
+                     */
+                    currentDate: {},
+
+                    /**
+                     * Current user's meal order.
+                     */
+                    order: {
+                        type: Array,
+                        statePath: 'currentOrder'
+                    },
+
+                    /**
+                     * User's history for the chosen day.
```

Comment:
Describe please inner structure in docs. (We shouldn't describe it multi times, so maybe let's describe the structure of properties of  components which are connected with redux)